pinot-controller/src/main/java/com/linkedin/pinot/controller/validation/ValidationManager.java

```diff
@@ -80,6 +80,7 @@ public ValidationManager(ValidationMetrics validationMetrics, PinotHelixResource
     _validationMetrics = validationMetrics;
     _pinotHelixResourceManager = pinotHelixResourceManager;
     _validationIntervalSeconds = config.getValidationControllerFrequencyInSeconds();
+    _autoCreateOnError = true;
```

Comment:
A misconfigured table during table creation is not a problem.

A table that has been misconfigured later, could end up trying to create a consuming segment every hour, but that is a smaller problem. There will be other alerts that will make sure that we correct the configuration.

In any case, I added a try/catch around the kafka/zk code, so that we can skip that partition after adding a warning and move on to other partitions that may warrant attention.

Lastly, I think we should implement the code to auto-create partitions when they stop consuming as well.

The servers should retry on timeout exception, and post a message to the controller on the kafkaoffset exception. If all replicas report kafka offset exception, then the controller can auto-create a new segment with a new offset.

I will add code for this soon.